Add doc on Telemetry API Tracing configuration (#9572)
istio/istio.io: content/en/docs/tasks/observability/distributed-tracing/configurability/mesh-and-proxy-config/index.md

---

> title: Using MeshConfig and Pod annotations

Reviewer: Fair enough

> title: Using MeshConfig and Pod annotations

Author: done.

> description: How to configure tracing options with the MeshConfig and pod annotations.

Author: How about 'Using' ?  The parent is 'Configuration', so repeating that causes stutter (and takes up space) in rendered page.

weight: 11
keywords: [telemetry,tracing]
aliases:
 - /docs/tasks/observability/distributed-tracing/configurability/
owner: istio/wg-policies-and-telemetry-maintainers
test: no
---

> {{< warning >}}

Author: switched to 'tip'

---
title: Configure tracing using MeshConfig and Pod annotations

> description: How to configure tracing options with the MeshConfig and pod annotations.

Reviewer: ```suggestion
description: How to configure tracing options using MeshConfig and pod annotations.
```

weight: 11
keywords: [telemetry,tracing]
aliases:
 - /docs/tasks/observability/distributed-tracing/configurability/
owner: istio/wg-policies-and-telemetry-maintainers
test: no
status: Beta/Experimental
---

{{< tip >}}

> Users are encouraged to transition to the [Telemetry API for tracing configuration](/docs/tasks/observability/distributed-tracing/configurability/telemetry-api/).

Reviewer 2: ```suggestion
Users are encouraged to transition to the [Telemetry API](/docs/tasks/observability/distributed-tracing/configurability/telemetry-api/) for tracing configuration.
```
